multicompany_property_delivery/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Creu Blanca
+# Copyright 2021 ForgeFlow, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Multi Company Stock",
```

Comment:
```suggestion
    "name": "Multi Company Delivery",
```
The `multicompany_property_stock` module already has the same name. There must have been a typing error when creating the module.

Reply:
@GuillemCForgeFlow done